ceph_cfg/presenter.py

```diff
@@ -349,3 +359,29 @@ def ceph_version(self):
 
     def cephfs_list(self):
         return self.model.cephfs_list
+
+
+    def df_osd(self, osd_number):
```

Comment:
This function is called in exactly one place, and besides the error checking has exactly two lines of data mangling. Does it make sense to fold into the calling place? (Also, osd_df calling df_osd is slightly confusing naming scheme ...)